shell/platform/common/accessibility_bridge.h

```diff
@@ -61,6 +61,27 @@ class AccessibilityBridge
   ///             To flush the pending updates, call the CommitUpdates().
   ///
   /// @param[in]  node           A pointer to the semantics node update.
+  void AddFlutterSemanticsNodeUpdate(const FlutterSemanticsNode2* node);
```

Comment:
While this isn't public API, these should really take a const ref rather than a const pointer, since this isn't an out param and null isn't an acceptable input.

Can you also update the old ones for consistency?

Reply:
+1 but this turned out to be pretty impactful (it affects all desktop embedders and unit tests). I'd like to keep this pull request focused on the embedder API changes, so I'll do the reference refactoring in a separate pull request and then rebase this one on top that.

Reply:
Awesome - even better.

Reply:
Here's the pull request for this refactoring: https://github.com/flutter/engine/pull/40167. I'll land that and then rebase this one :)

Reply:
Rebased!